en/publish/publish-vivo-instant-games.md

```diff
@@ -1,140 +1,146 @@
 
-  1、构建时，勾选 **小包模式**，填写 **小包模式服务器路径**。然后点击 **构建**。
+  1. When building, check the **packet mode** and fill in the **packet mode server path**. Then click on **Build**.
 
-  2、构建完成后，点击 **发布路径** 后面的 **打开** 按钮，将发布路径下的 **jsb-link/res** 目录上传到小包模式服务器。例如：默认发布路径是 build，则需要上传 build/jsb-link/res 目录。
+  2. After the build is complete, click the **Open** button after the **Publish Path** to upload the **jsb-link/res** directory under the release path to the packet mode server. For example, if the default publishing path is build, you need to upload the **build/jsb-link/res** directory.
 
-  此时，构建出来的 qgame 目录下将不再包含 res 目录，res 目录里的资源将通过网络请求从填写的 **小包模式服务器地址** 上下载。
+  At this point, the built-up `qgame` directory will no longer contain the `res` directory, and the resources in the `res` directory will be downloaded from the filled **packet mode server address** via the network request.
 
-二、**构建发布** 面板的相关参数设置完成后，点击 **构建**。构建完成后点击 **发布路径** 后面的 **打开** 按钮来打开构建发布包，可以看到在默认发布路径 build 目录下生成了 **qgame** 目录，该目录就是导出的 vivo 小游戏工程目录。
+Second, **Build release** After the relevant parameters of the panel are set, click **Build**. After the build is complete, click the **Open** button behind the **Publish Path** to open the build release package. You can see that the **qgame** directory is generated under the default release path build directory, which is the exported __Vivo Mini Game__ project directory.
 
 ![](./publish-vivo-instant-games/package.png)
 
-三、生成 rpk 包（如果在 **构建发布** 面板有填写了 **本地 npm 安装路径**，就可以跳过该步骤）。
+Third, generate the rpk package (if you have filled out the **local npm installation path** in the **Build Release** panel, you can skip this step).
 
 ```bash
-# 命令行指定到 qgame 目录下
+# Command line specified to the qgame directory
 cd E:\workspace\YourProject\build\qgame
 npm install
 npm run build
 ```
 
-然后在 /build/qgame/dist 目录下就会生成 rpk 包。
+Then the rpk package will be generated in the **/build/qgame/dist** directory.
 
 ![](./publish-vivo-instant-games/rpk.png)
 
-四、将打包出来的 rpk 运行到手机上。有以下两种方式可将 rpk 运行到手机上：
+4. Run the packaged rpk to your phone. There are two ways to run rpk on your phone:
 
-- **方法一**：
+- **method one**:
 
-    利用 vivo 小游戏打包工具命令生成网址和二维码
+    Generate URLs and QR codes using the __Vivo__ game packer command
 
     ```bash
-    # 先把命令行指定到 qgame 目录下
+    # First assign the command line to the qgame directory
     cd E:\workspace\YourProject\build\qgame
-    # 生成网址和二维码
+    # Generate URL and QR code
     npm run server
     ```
 
-    然后在 Android 设备上打开之前已经安装完成的 **快应用 & vivo 小游戏调试器**
+    Then open the **Quick app & vivo game debugger that was installed before on your Android device**
```

Comment:
Then open the **Quick App & Vivo Mini Game Debugger** that was installed before on your Android device